src/components/OrgChart.vue

```diff
@@ -0,0 +1,164 @@
+
+					return `
+						<div style="padding-top:30px;background-color:none;margin-left:1px;height:${d.height}px;border-radius:2px;overflow:visible">
```

Comment:
is there any chance for d3 to work with Vue components instead of string concatenation so that we can safely prevent user content injection into the DOM?